virttest/qemu_storage.py

```diff
@@ -345,10 +346,10 @@ def snapshot_del(self, blkdebug_cfg=""):
                                        " parameters")
         if blkdebug_cfg:
             cmd += " blkdebug:%s:%s" % (blkdebug_cfg, self.image_filename)
+            return process.run(cmd, ignore_status=True)
```

Comment:
as i commented in the  autotest/tp-qemu#654, you misunderstand the function of this patch.